config/config.go

```diff
@@ -567,6 +571,14 @@ func NewConfig() *Config {
 	flag.Var(cfg.LuaModules, "lua-modules", "comma separated list of lua filter modules. Use <module>.<symbol> to selectively enable module symbols, for example: package,base._G,base.print,json")
 	flag.Var(cfg.LuaSources, "lua-sources", `comma separated list of lua input types for the lua() filter. Valid sources "", "file", "inline", "file,inline" and "none". Use "file" to only allow lua file references in lua filter. Default "" is the same as "file","inline". Use "none" to disable lua filters.`)
 
+	// Passive Health Checks
+	flag.BoolVar(&cfg.EnablePassiveHealthCheck, "enable-passive-health-check", false, "enables passive health check")
+	cfg.PassiveHealthCheckParams = mapFlags{values: map[string]string{
+		"total-requests-threshold":           "10",
+		"timed-out-requests-ratio-threshold": "0.1",
```

Comment:
Why is it time out and not failed?

Reply:
Yeah, naming is hard. I came up to `failed-round-trips-ratio-threshold` as the best option I could propose.

Reply:
We can call it `failed-ratio-threshold`.